TheMET/TheMET/ViewControllers/CategoryViewController.swift

```diff
@@ -8,18 +8,140 @@
 import Foundation
 import UIKit
 
-class CategoryViewController: UIViewController {
+class CategoryViewController: UIViewController, UITableViewDelegate, UITableViewDataSource  {
     
-    @IBOutlet
-    private var cancelButon: UIButton!
+    var titleCategoria: String = String()
+
+    private let favoriteService = FavoritesService.standart
+    
+    private let loadingCategoryView = LoadingPlaceholderView.constructView(configuration: .categoryArtworksLoading)
+    private let failedCategoryView = LoadingPlaceholderView.constructView(configuration: .categoryFailed)
+    
+    private let categoryTableView: UITableView = UITableView(frame: .zero, style: .plain)
+    
+    enum Status {
+        case loaded
+        case loading
+        case failed
+    }
+    
+    var contentStatus: Status = .loading {
+        didSet {
+            self.updateContent()
+        }
+    }
+    
+    override func awakeFromNib() {
```

Comment:
This method override doesn't do anything

Reply:
+